picky-signtool/src/config.rs

```diff
@@ -134,6 +135,15 @@ pub fn config() -> ArgMatches<'static> {
                 .requires_all(&[ARG_SIGN, ARG_CERTFILE])
                 .display_order(7),
         )
+        .arg(Arg::with_name(ARG_TIMESTAMP)
+            .short("t")
+            .long(ARG_TIMESTAMP)
+            .value_name("URL")
+            .help("Specify url of a Authenticode TSA server to timestamp")
```

Comment:
Just for the sake of clarity:
```suggestion
            .help("Specify url of a Authenticode TSA server when timestamping")
```